proto/valorem/trade/v1/auth.proto

```diff
@@ -2,7 +2,7 @@ syntax = "proto3";
 
 import "types.proto";
 
-package quay;
+package valorem.trade.v1;
```

Comment:
Doesn't the path already imply this? How does the FE compile the proto given you have to provide a path for the build.rs I'm not sure if the change in package name really does anything